apps/cards.py

```diff
@@ -502,10 +495,37 @@ def card_id(pdf):
                         ],
                     ),
                     dmc.AccordionPanel(
-                        [
-                            download_tab,
-                            dcc.Markdown('See {}/api for more options'.format(APIURL)),
-                        ],
+                        html.Div(
+                            [
+                                dmc.Group(
+                                    [
+                                        dmc.Button(
+                                            "JSON",
+                                            id='download_json',
+                                            variant="outline",
+                                            color='indigo',
+                                            leftIcon=[DashIconify(icon="mdi:code-json")],
+                                        ),
+                                        dmc.Button(
+                                            "VOTable",
+                                            id='download_votable',
+                                            variant="outline",
+                                            color='indigo',
+                                            leftIcon=[DashIconify(icon="mdi:xml")],
+                                        ),
+                                        html.Div(objectid, id='download_objectid', className='d-none'),
+                                        html.Div(APIURL, id='download_apiurl', className='d-none'),
+                                    ], position="center"
+                                ),
+                                help_popover(
```

Comment:
Instead of having the helper here, can we add a third button labeled for example 'Code snippet' (propose something better if you find!) that actually opens the popover?

![Screenshot from 2023-12-04 08-40-59](https://github.com/astrolabsoftware/fink-science-portal/assets/20426972/3390f1a2-6ffd-4a32-8981-5444124a1cd6)

Reply:
Committed a fix that makes this a button. For readability, saving space on smaller screens, and to better distinguish from download formats, it has just a help icon, with no text.
<img width="361" alt="Screenshot 2023-12-04 at 13 48 25" src="https://github.com/astrolabsoftware/fink-science-portal/assets/6555548/1d256922-fad8-414f-83ee-41911b88a373">